README.md

```diff
@@ -1,12 +1,10 @@
 # smt
 
-A Go library that implements a Sparse Merkle tree for a key-value map.
+A Go library that implements a Sparse Merkle tree for a key-value map. The tree implements the same optimisations specified in the [Libra whitepaper](https://developers.libra.org/docs/assets/papers/the-libra-blockchain/2020-05-26.pdf), to reduce the number of hash operations required per tree operation to O(k) where k is the number of non-empty elements in the tree.
```

Comment:
Not entirely right? (I mean the IO optimizations aka "nibbles" / reducing the io ops per inner node)